src/blueprints/tv2/timeline-object-factories/tv2-caspar-cg-timeline-object-factory.ts

```diff
@@ -30,6 +31,8 @@ enum CasparCgSlot {
   UNKNOWN = 'UNKNOWN'
 }
 
+const HTML_GRAPHICS_INDEX_FILE: string = 'index'
```

Comment:
CasparCG supports not having an extension, but for clarity: Should the index filename have `.html` suffixed?
And this is a filename, not a file :)

Reply:
This is meant to be the name of the file that CasparCG should chose to display HTML graphics
![image](https://github.com/tv2/sofie-server/assets/23049647/c00009cd-f822-444c-ae5d-63e6e14779a3)
As far as I'm aware, we are not supposed to send the file type with, but I might be wrong.

I have updated the variable name to indicate that it's the filename and not the file.

Reply:
Fair, no extension then :)